domain/src/main/java/camp/nextstep/edu/tictactoe/domain/Blocks.kt

```diff
@@ -0,0 +1,57 @@
+package camp.nextstep.edu.tictactoe.domain
+
+class Blocks(private val blockSize: Int) {
+
+    private val blocks = Array(blockSize) { Array(blockSize) { Block.BlockNon } }
+
```

Comment:
다음을 참고하여 Blocks를 일급 컬렉션으로 만들어보면 어떨까요? 🙂 

https://jojoldu.tistory.com/412

```suggestion
class Blocks(private val blocks: Array<Block>) {

    constructor(blockSize: Int) : this(Array(blockSize) { Block.BlockNon })
    
```